src/pkg/inspectors/imagescanner/types/export_doc.go

```diff
@@ -0,0 +1,35 @@
+package types
+
+type ContainerInfo struct {
+	ContainerId      string `json:"containerId"`
+	ContainerName    string `json:"containerName"`
+	ContainerImage   string `json:"containerImage"`
+	ContainerImageId string `json:"containerImageId"`
```

Comment:
Things in this file are copied from a removed file in which the stuct of each scanner is defined there.